pkg/chunkserver/chunkserver.go

```diff
@@ -106,14 +107,19 @@ func (c *Cluster) Start(nodeNameIP map[string]string) error {
 	logger.Info("create physical pool successed")
 
 	// 3. startChunkServers start all chunkservers for each device of every node
-	err = c.startChunkServers()
+	var chunkServers []*v1.Deployment
```

Comment:
I think this logic should not be added here. All processing is completed in startChunks, and the startChunkServers function should be that every deployment has started successfully. 

https://github.com/opencurve/curve-operator/blob/ebde98e0ef8d203c9eea8f78f44b2b78ab5ee14b/pkg/chunkserver/spec.go#L64